_config.yml

```diff
@@ -91,6 +91,9 @@ collections:
   external_links:
     permalink: /:collection/:path/
     output: true
+  integrations:
```

Comment:
Not a technical requirement, but this ideally match the order in which we the new section appears in the side-nav? Going by line 115, this should appear under dashboards, starting at line 43.

Reply:
@Naarcha-AWS The team wants it to be a top-level section. Are you suggesting it follow the OpenSearch Dashboards top-level section?

Reply:
No. Just moving this line up to below the Dashboards section on the collection list.

Reply:
Got it. Agreed.